gatsby/onCreateNode.js

```diff
@@ -9,6 +9,22 @@
 // Parse date information out of blog post filename.
 const BLOG_POST_FILENAME_REGEX = /([0-9]+)\-([0-9]+)\-([0-9]+)\-(.+)\.md$/;
 
+function buildRedirectString(permalink, redirect_from) {
+  if (!permalink || permalink.indexOf('.html') === -1) {
```

Comment:
Instead of indexOf, can we do `!permalink.endsWith('.html')` so it won't catch if there is .html elsewhere in the name?

Reply:
Note: `endsWith()` isn't supported by IE unless we polyfill it

Reply:
I sorta assumed this runs at build time. If not, `!/\.html$/.test(permalink)` would work.

Reply:
Oops. I think you're right

Reply:
Thanks, will do that!